erpnext/templates/pages/help.py

```diff
@@ -5,22 +5,74 @@
 
 def get_context(context):
 	context.no_cache = 1
+	context.bg = 'background-color: #fafbfc; border-radius:0'
+	context.align_greeting = 'start'
+	context.align_search_box = '0'
 	settings = frappe.get_doc("Support Settings", "Support Settings")
 	s = settings
 
+	context.greeting_text = s.greeting_text if s.greeting_text else "We're here to help"
```

Comment:
```suggestion
	context.greeting_text = s.greeting_text or _("We're here to help")
```

Translation required

Reply:
OK will resolve this